src/univariate/continuous/noncentralbeta.jl

```diff
@@ -35,11 +35,22 @@ partype(::NoncentralBeta{T}) where {T} = T
 
 ### Evaluation & Sampling
 
-# TODO: add mean and var
+function mean(d::NoncentralBeta)
+    α, β, λ = params(d)
+    return α / (α + β) * exp(-λ / 2) * pFq([1 + α, α + β], [α, α + β + 1], λ / 2)
```

Comment:
I wonder if this can be simplified further, given e.g. the formula in https://en.wikipedia.org/wiki/Generalized_hypergeometric_function#Generalization_of_Kummer's_transformations_and_identities_for_2F2. It seems, however, that the first identity leads to incorrect results here since one would end up with just `alpha / (alpha  + beta)` - I guess there is some assumption on a, b, c, and d that is not satisfied here.

(Not our fault but also a bit annoying that one has to call `pFq` with vectors instead of tuples)

Reply:
[Wolfram](https://reference.wolfram.com/language/ref/NoncentralBetaDistribution.html) is the reference I used. Can you think of any edge case where the exception might be triggered?